### ISSUE TITLE
Remove the photo credit on the bottom from UnSplash

### ISSUE THREAD

**Issue by KJpi5** [delightedCrow/WallberryTheme#51]
Is there a way to remove the "Photo by Joe Blow on Unsplash"??

I know credit should be given on things but I dont want it being on my home project that no one else but me will see.

**Comment by lopuk** [comment]
Theoretically, you could modify the file '**/opt/mm/mounts/modules/WallberryTheme/WallberryTheme. njk**', but it's nice to credit the creators of the things we enjoy.

**Comment by delightedCrow** [comment]
yeah it doesn’t sit right with me to remove the photo credit, which is the one condition the artists making these images available for free are asking for in return.

like @lopuk says above, not hard to make an edit to the template files if you really want to do it for your own, but I won't add it to the project
